app/ceom/maps/templates/maps/pixels_validation.html

```diff
@@ -0,0 +1,158 @@
+{% extends "maps/base.html" %}
+
+{% load i18n %}
+{% block sidebar_extra %}
+<nav class="card bg-light ml-5 mt-3 shadow-sm">
+    {% include "photos/search_form.html" %}
+</nav>
+{% endblock %}
+
+{% block content %}
+<script defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBo98LuzUr-aVpxRZPFHkBl2m4ogHeROGQ&callback=initMap" type="text/javascript"></script>
+<style>
+    .menu-field {
+        font-size: small;
+        height: 22px;
+        padding-top: 0;
+        padding-bottom: 0;
+    }
+
+    .control-group {
+        display: inline-flex;
+    }
+
+    .font {
+        font-size: 9pt;
+        color: #555;
+    }
+
+    .title {
+        color: #0f4ba4;
+    }
+
+    .batch {
+        width: initial;
+        font-size: small;
+    }
+    #map {
+           height: 768px;
+    }
+    @media screen and (min-width:575px) and (max-width: 1250px) {
+
+        .title {
+            font-size: min(max(3.9vw, 10px), 30px);
+        }
+        .content {
+            padding-left: 5%;
+        }
+    }
+
+    @media screen and (max-width:575px) {
+
+        .title {
+            font-size: min(max(3.8vw, 23px), 30px);
+        }
+        .content {
+            padding-left: 5%;
+        }
+    }    
+</style>
+<nav>
+    <h2 class="legend title">Pixels Validation</h2>
+    <p style="font-size:large">Here you can retrieve geographical data of pixels in the selected area. The file of data you get is in *.csv file</p>
+    <div class="my-2">
+        <p><strong style="font-size:large"> 1. Select size of region. </strong></p>
+        <ul style="list-style: none;">
+            <li>
+                <p><input type="radio" name="boxSize" value="5" checked> 5 degrees</p>
+            </li>
+            <li>
+                <p><input type="radio" name="boxSize" value="10"> 10 degrees</p>
+            </li>
+        </ul>
+        <p><strong style="font-size:large">2. Zoom-in the map and then click on a point of interest </strong></p>
+        <p>
+            <div id="map"></div>
+            <div class="row">
+                <div class="col-sm-3">
+                    <input type="checkbox" id="auto_zoom" name="auto_zoom" checked /> Zoom to selected region</div>
+            </div>
+        </p> 
+        <p><strong style="font-size:large">3. Check your request and click Download</strong></p>
+        <div>
+            <button class="btn btn-success mb-2" onclick="pixels_validation_csv()"><i class="icon-plus-sign "></i>Download CSV</button>
+        </div>
+    </div>
+</nav>
+<script>
+    let map;
+    let polygon;
+    let lat = 0;
+    let lon = 0;
+    let centerLocation = {
+        lat: lat,
+        lng: lon
+    };
+    var size = 5;
+	function initMap() 
```

Comment:
Strange indentation here
Put a blank line between your variables being declared and the start of the initMap function
And fix the indentation of initMap

It might be a whitespace difference - if it looks normal in VSCode let me know and I will show you how to fix it

Reply:
Yes, the indentation of initMap looks normal in VSCode